app/javascript/packs/document-capture.tsx

```diff
@@ -120,6 +128,7 @@ const App = composeComponents(
         inPersonOutageMessageEnabled: inPersonOutageMessageEnabled === 'true',
         inPersonOutageExpectedUpdateDate,
         inPersonFullAddressEntryEnabled: inPersonFullAddressEntryEnabled === 'true',
+        optedInToInPersonProofing: optedInToInPersonProofing === 'true',
```

Comment:
If we made this a string that represented the value of the user selection, we wouldn't have to coerce to a boolean here, I think?

Reply:
I think the value in JS should absolutely be a boolean like it is here, it gets serialized to a string via a `data-` attribute and converted just like the other properties. Doing this conversion lets us have stricter typing in Typescript as well.

I'm unsure what the implications are for the blob stored in the Ruby session, but this can be independent of that

Reply:
Cool! Thanks @zachmargolis